packages/bus-rdf-update-quads/lib/IQuadDestination.ts

```diff
@@ -21,6 +21,12 @@ export interface IQuadDestination {
    * @return {AsyncIterator<RDF.Quad>} The deleted quad stream.
    */
   delete: (quads: AsyncIterator<RDF.Quad>) => Promise<void>;
+  /**
+   * Updates destination with quad stream to insert and quad stream to delete.
+   * @param quadStreamInsert The quads to insert.
+   * @param quadStreamDelete The quads to delete.
+   */
+  update: (quadStreamInsert: AsyncIterator<RDF.Quad>, quadStreamDelete: AsyncIterator<RDF.Quad>) => Promise<void>;
```

Comment:
Perhaps we can even remove the `insert` and `delete` methods, and specify `update` as having insert and delete streams each as optional.

I would also propose being able to call it as follows: `update({ insert: streamA, delete: streamB })`. (just so the order of args is not arbitrary)